test/integration/test_expid.py

```diff
@@ -110,3 +134,127 @@ def test_copy_minimal(has_min_yaml, autosubmit_exp, autosubmit):
                     description="Pytest experiment")
             assert exc_info.value.code == 7011
             assert "minimal.yml" in str(exc_info.value)
+
+def test_create_expid_default_hpc(autosubmit: Autosubmit) -> None:
+    """Create expid with the default hcp value (no -H flag defined).
+
+    .. code-block:: console
+
+        autosubmit expid -d "test descript"
+
+    :param autosubmit: Autosubmit interface that instantiate with no experiment.
+    :type autosubmit: Autosubmit
+
+    :return: None
+    """
+    # create default expid
+    platform_name = "local"
+    experiment_id = autosubmit.expid(
+        'experiment_id',
+        platform_name,
+        minimal_configuration=True
+    )
+
+    # capture the platform using the "describe"
+    describe = autosubmit.describe(experiment_id)
+    hpc_result  = describe[4].lower()
+
+    assert hpc_result == platform_name
+
+@pytest.mark.parametrize("fake_hpc, expected_hpc", [
+    ("mn5", "mn5"),
+    ("", "local"), ])
+def test_create_expid_flag_hpc(fake_hpc: str, expected_hpc: str, autosubmit: Autosubmit) -> None:
+    """Create expid using the flag -H. Defining a value for the flag and not defining any value for that flag.
+
+    .. code-block:: console
+
+        autosubmit expid -H ithaca -d "experiment is about..."
```

Comment:
s/ithaca/mn5

And description here is "experiment is about..." but in the code is "experiment" (nit-pick, doesn't really matter)